packages/zen-observable-ts/src/zenObservable.ts

```diff
@@ -0,0 +1,69 @@
+declare function require(name: string);
+const _Observable = require('zen-observable');
+
+import { ZenObservable } from './types';
+
+export { ZenObservable };
+
+export type Observer<T> = ZenObservable.Observer<T>;
+export type Subscriber<T> = ZenObservable.Subscriber<T>;
+export type ObservableLike<T> = ZenObservable.ObservableLike<T>;
+
+export default class Observable<T> {
+  private observable: any;
+
+  public static from<R>(
+    observable: Observable<R> | ZenObservable.ObservableLike<R> | ArrayLike<R>,
+  ): Observable<R> {
+    return _Observable.from(observable);
+  }
+
+  public static of<R>(...items: R[]): Observable<R> {
+    return _Observable.of(...items);
+  }
+
+  constructor(subscriber: ZenObservable.Subscriber<T>) {
+    this.observable = new _Observable(subscriber);
+  }
+
+  public subscribe(
+    observerOrNext: ((value: T) => void) | ZenObservable.Observer<T>,
+    error?: (error: any) => void,
+    complete?: () => void,
+  ): ZenObservable.Subscription {
+    return this.observable.subscribe(observerOrNext, error, complete);
+  }
+
+  public forEach(fn: (value: T) => void): Promise<void> {
+    return this.observable.forEach(fn);
```

Comment:
If `Observable<T>` extends `_Observable`, then methods implementations like this would become
```typescript
return super.forEach(fn);
```
because `this instanceof _Observable`.